realm/realm-library/src/main/java/io/realm/RealmQuery.java

```diff
@@ -522,31 +521,31 @@ public boolean isValid() {
      * In comparison. This allows you to test if objects match any value in an array of values.
      *
      * @param fieldName the field to compare.
-     * @param values array of values to compare with and it cannot be null or empty.
+     * @param values array of values to compare with. If {@code null} or the empty is parsed in the query will never
```

Comment:
If {@ null} or empty is passed, the query will never ...

_same  remark for other methods_